components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -107,9 +107,9 @@ internal sealed class SubscriptionWrapper : ISubscription {
 
             /// <inheritdoc/>
             public async Task CloseAsync() {
-                await _lock.WaitAsync();
+                await _lock.WaitAsync().ConfigureAwait(false);
                 try {
-                    _logger.Information("Closing subscription {subscription}", Id);
```

Comment:
>Information [](start = 28, length = 11)

I would generally prefer to keep these kind of logs more visible. They will only happen once during runtime so they are not that chatty. On the other hand, if something goes wrong and everything is verbose you have no chance follow any hind of flow and to help the customer.

Reply:
In a test run with 75 servers/90 nodes, in less than 10 min it generated this amount:
![image](https://user-images.githubusercontent.com/1353540/83923932-91fc3200-a783-11ea-8511-32194edbf538.png)
I can revert if you insist :)

Reply:
typically when subscriptions/sessions get closed you're loosing data. If we do not trace such things the customer will only notice he lost data, but will have no clue why. and we will have to answer why ...